index.bs

```diff
@@ -168,6 +168,11 @@ div.enum-description > table > thead > tr > th {
       </dd>
     </dl>
 
+      If the constructor which authors pass is not a constructor of
+      {{HTMLMediaElement}} (or its extension, such as {{HTMLVideoElement}} and
+      {{HTMLAudioElement}}) or {{AudioContext}}, then these methods will throw a
+      {{TypeError}}.
```

Comment:
I'm not clear whether the methods will throw a `TypeError` because:
1. Web IDL says to throw a `TypeError` in such cases; or
2. This spec tells implementers to throw a `TypeError`.

If the former (and I suspect that is the case), I would wrap that paragraph in a `<div class="note">` to make it informative-only as we're not adding normative requirements on top of what Web IDL mandates and probably don't want to duplicate them either.

If the latter, I would reformulate the paragraph to make it clear that we're asking implementers to validate the given constructor and throw an error. In other specs, this is typically done by using a more algorithmic step-by-step approach as in "When the getAutoplayPolicy(ctor) method is called, the user agent MUST run the following steps: 1. If ctor is not a constructor of HTMLMediaElement then throw a TypeError". This reformulation can no doubt be done separately and later on.

Reply:
Yes, I meant the exception thrown by Web IDL definition. Will modify that, thank you!